assets/css/calypsoify.css

```diff
@@ -1350,4 +1350,44 @@ p.search-box {
 .subsubsub .search-box.is-expanded .search-box__close-icon,
 .subsubsub .search-box.is-expanded input[type="search"] {
 	display: flex;
+}
+
+/* Fix display of the "Please copy and paste this information in your ticket when contacting support" notice displayed on the system status page */
+.woocommerce_page_wc-status .woocommerce-message a.debug-report {
+	color: #0073aa !important;
+	padding-bottom: 16px !important;
+}
+
+.woocommerce_page_wc-status .woocommerce-message a.docs, .woocommerce_page_wc-status .woocommerce-message a.docs:hover {
+	color: #0073aa !important;
+	border: 0;
+	padding: 0;
+	font-size: 14px !important;
+	font-weight: 400 !important;
+}
+
+
+.woocommerce_page_wc-status .woocommerce-message #copy-for-support {
+	color: #0073aa !important;
+	border: 0;
+	padding: 0;
+	font-size: 14px !important;
+	font-weight: 400 !important;
+}
+
+.woocommerce_page_wc-status .woocommerce-message a.debug-report:hover,
+.woocommerce_page_wc-status .woocommerce-message a.docs:hover,
+.woocommerce_page_wc-status .woocommerce-message #copy-for-support:hover {
+	color: #00a0d2 !important;
+}
+
+.woocommerce_page_wc-status .woocommerce-message.inline {
```

Comment:
Should we remove box-shadow and rounded corners on this to match the other cards on this page?

```css
    box-shadow: none;
    border-radius: 0;
```